src/Common/Core/Twig/BaseTwigTemplate.php

```diff
@@ -9,10 +9,10 @@
 use SpoonForm;
 use Symfony\Bridge\Twig\Form\TwigRendererEngine;
 use Symfony\Bundle\TwigBundle\TwigEngine;
+use Symfony\Component\Stopwatch\Stopwatch;
```

Comment:
Where did this come from?

Reply:
From debugging it :D should be removed

Reply:
Very well, remove it and i'll approve this PR